src/css/components/deployments.less

```diff
@@ -12,3 +12,10 @@
     width: 100%;
   }
 }
+
+.deployments {
+  td {
+    padding-bottom: @base-spacing-unit !important;
```

Comment:
Yes let's do all we can to remove `!important` – it's a maintainance time-bomb waiting to blow up in our faces :D